[2022-04-11 16:46:51,029] {processor.py:163} INFO - Started process (PID=11150) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:46:51,031] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:46:51,031] {logging_mixin.py:109} INFO - [2022-04-11 16:46:51,031] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:46:51,130] {logging_mixin.py:109} INFO - [2022-04-11 16:46:51,128] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 2, in <module>
    import cvxpy as cvx
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/cvxpy/__init__.py", line 18, in <module>
    import cvxpy.interface.scipy_wrapper
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/cvxpy/interface/__init__.py", line 17, in <module>
    from cvxpy.interface.matrix_utilities import *
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/cvxpy/interface/matrix_utilities.py", line 20, in <module>
    import scipy.sparse as sp
ModuleNotFoundError: No module named 'scipy'
[2022-04-11 16:46:51,140] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:46:51,151] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 0.125 seconds
[2022-04-11 16:47:30,013] {processor.py:163} INFO - Started process (PID=11350) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:47:30,014] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:47:30,015] {logging_mixin.py:109} INFO - [2022-04-11 16:47:30,015] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:47:30,086] {logging_mixin.py:109} INFO - [2022-04-11 16:47:30,084] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 2, in <module>
    import cvxpy as cvx
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/cvxpy/__init__.py", line 18, in <module>
    import cvxpy.interface.scipy_wrapper
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/cvxpy/interface/__init__.py", line 17, in <module>
    from cvxpy.interface.matrix_utilities import *
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/cvxpy/interface/matrix_utilities.py", line 20, in <module>
    import scipy.sparse as sp
ModuleNotFoundError: No module named 'scipy'
[2022-04-11 16:47:30,093] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:47:30,103] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 0.092 seconds
[2022-04-11 16:48:11,042] {processor.py:163} INFO - Started process (PID=11563) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:48:11,046] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:48:11,046] {logging_mixin.py:109} INFO - [2022-04-11 16:48:11,046] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:48:12,650] {logging_mixin.py:109} INFO - [2022-04-11 16:48:12,650] {font_manager.py:1443} INFO - generated new fontManager
[2022-04-11 16:48:14,341] {logging_mixin.py:109} INFO - [2022-04-11 16:48:14,341] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:48:14,342] {logging_mixin.py:109} INFO - [2022-04-11 16:48:14,341] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = requests_cache.CachedSession(cache_name='cache', backend='sqlite')
AttributeError: module 'requests_cache' has no attribute 'CachedSession'
[2022-04-11 16:48:14,348] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:48:14,387] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.353 seconds
[2022-04-11 16:49:04,507] {processor.py:163} INFO - Started process (PID=11827) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:49:04,510] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:49:04,511] {logging_mixin.py:109} INFO - [2022-04-11 16:49:04,511] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:49:10,865] {logging_mixin.py:109} INFO - [2022-04-11 16:49:10,865] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:49:10,866] {logging_mixin.py:109} INFO - [2022-04-11 16:49:10,865] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = requests_cache.CachedSession(cache_name='cache', backend='sqlite')
AttributeError: module 'requests_cache' has no attribute 'CachedSession'
[2022-04-11 16:49:10,874] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:49:10,886] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 6.383 seconds
[2022-04-11 16:49:52,168] {processor.py:163} INFO - Started process (PID=12094) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:49:52,170] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:49:52,170] {logging_mixin.py:109} INFO - [2022-04-11 16:49:52,170] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:49:54,088] {logging_mixin.py:109} INFO - [2022-04-11 16:49:54,088] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:49:54,089] {logging_mixin.py:109} INFO - [2022-04-11 16:49:54,089] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = requests_cache.CachedSession(cache_name='cache', backend='sqlite')
AttributeError: module 'requests_cache' has no attribute 'CachedSession'
[2022-04-11 16:49:54,095] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:49:54,105] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 1.939 seconds
[2022-04-11 16:50:44,718] {processor.py:163} INFO - Started process (PID=12406) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:50:44,719] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:50:44,720] {logging_mixin.py:109} INFO - [2022-04-11 16:50:44,720] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:50:46,890] {logging_mixin.py:109} INFO - [2022-04-11 16:50:46,890] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:50:46,891] {logging_mixin.py:109} INFO - [2022-04-11 16:50:46,891] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = requests_cache.CachedSession(cache_name='cache', backend='sqlite')
AttributeError: module 'requests_cache' has no attribute 'CachedSession'
[2022-04-11 16:50:46,897] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:50:46,906] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.190 seconds
[2022-04-11 16:51:38,091] {processor.py:163} INFO - Started process (PID=12672) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:51:38,094] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:51:38,095] {logging_mixin.py:109} INFO - [2022-04-11 16:51:38,095] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:51:40,423] {logging_mixin.py:109} INFO - [2022-04-11 16:51:40,423] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:51:40,424] {logging_mixin.py:109} INFO - [2022-04-11 16:51:40,423] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = requests_cache.CachedSession(cache_name='cache', backend='sqlite')
AttributeError: module 'requests_cache' has no attribute 'CachedSession'
[2022-04-11 16:51:40,430] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:51:40,440] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.353 seconds
[2022-04-11 16:52:18,632] {processor.py:163} INFO - Started process (PID=12738) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:52:18,633] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:52:18,634] {logging_mixin.py:109} INFO - [2022-04-11 16:52:18,634] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:52:20,856] {logging_mixin.py:109} INFO - [2022-04-11 16:52:20,855] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:52:20,856] {logging_mixin.py:109} INFO - [2022-04-11 16:52:20,856] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = requests_cache.CachedSession(cache_name='cache', backend='sqlite')
AttributeError: module 'requests_cache' has no attribute 'CachedSession'
[2022-04-11 16:52:20,862] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:52:20,872] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.242 seconds
[2022-04-11 16:53:13,049] {processor.py:163} INFO - Started process (PID=12881) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:53:13,051] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:53:13,052] {logging_mixin.py:109} INFO - [2022-04-11 16:53:13,052] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:53:15,495] {logging_mixin.py:109} INFO - [2022-04-11 16:53:15,495] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:53:15,496] {logging_mixin.py:109} INFO - [2022-04-11 16:53:15,495] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = requests_cache.CachedSession(cache_name='cache', backend='sqlite')
AttributeError: module 'requests_cache' has no attribute 'CachedSession'
[2022-04-11 16:53:15,503] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:53:15,514] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.472 seconds
[2022-04-11 16:53:59,007] {processor.py:163} INFO - Started process (PID=12953) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:53:59,008] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:53:59,009] {logging_mixin.py:109} INFO - [2022-04-11 16:53:59,009] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:54:00,996] {logging_mixin.py:109} INFO - [2022-04-11 16:54:00,996] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:54:00,997] {logging_mixin.py:109} INFO - [2022-04-11 16:54:00,996] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = requests_cache.CachedSession(cache_name='cache', backend='sqlite')
AttributeError: module 'requests_cache' has no attribute 'CachedSession'
[2022-04-11 16:54:01,004] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:54:01,015] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.010 seconds
[2022-04-11 16:54:39,768] {processor.py:163} INFO - Started process (PID=13015) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:54:39,769] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:54:39,770] {logging_mixin.py:109} INFO - [2022-04-11 16:54:39,770] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:54:42,154] {logging_mixin.py:109} INFO - [2022-04-11 16:54:42,154] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:54:42,156] {logging_mixin.py:109} INFO - [2022-04-11 16:54:42,155] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = requests_cache.CachedSession(cache_name='cache', backend='sqlite')
AttributeError: module 'requests_cache' has no attribute 'CachedSession'
[2022-04-11 16:54:42,168] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:54:42,185] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.420 seconds
[2022-04-11 16:55:33,060] {processor.py:163} INFO - Started process (PID=13158) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:55:33,063] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:55:33,064] {logging_mixin.py:109} INFO - [2022-04-11 16:55:33,064] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:55:35,970] {logging_mixin.py:109} INFO - [2022-04-11 16:55:35,970] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:55:35,971] {logging_mixin.py:109} INFO - [2022-04-11 16:55:35,971] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = requests_cache.core.CachedSession(cache_name='cache', backend='sqlite')
AttributeError: module 'requests_cache' has no attribute 'core'
[2022-04-11 16:55:35,978] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:55:35,989] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.933 seconds
[2022-04-11 16:55:51,766] {processor.py:163} INFO - Started process (PID=13270) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:55:51,767] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:55:51,768] {logging_mixin.py:109} INFO - [2022-04-11 16:55:51,768] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:55:54,268] {logging_mixin.py:109} INFO - [2022-04-11 16:55:54,268] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:55:54,269] {logging_mixin.py:109} INFO - [2022-04-11 16:55:54,268] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = requests_cache.core.CachedSession(cache_name='cache', backend='sqlite')
AttributeError: module 'requests_cache' has no attribute 'core'
[2022-04-11 16:55:54,279] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:55:54,294] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.532 seconds
[2022-04-11 16:56:22,327] {processor.py:163} INFO - Started process (PID=13295) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:56:22,329] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:56:22,329] {logging_mixin.py:109} INFO - [2022-04-11 16:56:22,329] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:56:24,888] {logging_mixin.py:109} INFO - [2022-04-11 16:56:24,887] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:56:24,889] {logging_mixin.py:109} INFO - [2022-04-11 16:56:24,888] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 50, in <module>
    session.headers = {'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:89.0) Gecko/20100101 Firefox/89.0',
NameError: name 'session' is not defined
[2022-04-11 16:56:24,896] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:56:24,910] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.586 seconds
[2022-04-11 16:56:36,048] {processor.py:163} INFO - Started process (PID=13371) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:56:36,052] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:56:36,055] {logging_mixin.py:109} INFO - [2022-04-11 16:56:36,054] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:56:41,518] {logging_mixin.py:109} INFO - [2022-04-11 16:56:41,516] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 48, in <module>
    ]
NameError: name 'session' is not defined
[2022-04-11 16:56:41,527] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:56:41,543] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 5.503 seconds
[2022-04-11 16:57:11,752] {processor.py:163} INFO - Started process (PID=13488) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:57:11,753] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:57:11,754] {logging_mixin.py:109} INFO - [2022-04-11 16:57:11,753] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:57:13,860] {logging_mixin.py:109} INFO - [2022-04-11 16:57:13,860] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:57:13,862] {logging_mixin.py:109} INFO - [2022-04-11 16:57:13,861] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = requests_cache.core.CachedSession(cache_name='cache', backend='sqlite')
AttributeError: module 'requests_cache' has no attribute 'core'
[2022-04-11 16:57:13,878] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:57:13,902] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.153 seconds
[2022-04-11 16:57:50,561] {processor.py:163} INFO - Started process (PID=13522) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:57:50,563] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:57:50,564] {logging_mixin.py:109} INFO - [2022-04-11 16:57:50,564] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:57:52,598] {logging_mixin.py:109} INFO - [2022-04-11 16:57:52,598] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:57:52,599] {logging_mixin.py:109} INFO - [2022-04-11 16:57:52,598] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = requests_cache.core.CachedSession(cache_name='cache', backend='sqlite')
AttributeError: module 'requests_cache' has no attribute 'core'
[2022-04-11 16:57:52,604] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:57:52,614] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.056 seconds
[2022-04-11 16:58:23,396] {processor.py:163} INFO - Started process (PID=13552) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:58:23,398] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:58:23,398] {logging_mixin.py:109} INFO - [2022-04-11 16:58:23,398] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:58:25,437] {logging_mixin.py:109} INFO - [2022-04-11 16:58:25,437] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:58:25,438] {logging_mixin.py:109} INFO - [2022-04-11 16:58:25,437] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = requests_cache.core.CachedSession(cache_name='cache', backend='sqlite')
AttributeError: module 'requests_cache' has no attribute 'core'
[2022-04-11 16:58:25,445] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:58:25,456] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.064 seconds
[2022-04-11 16:59:01,227] {processor.py:163} INFO - Started process (PID=13586) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:59:01,228] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:59:01,229] {logging_mixin.py:109} INFO - [2022-04-11 16:59:01,229] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:59:03,249] {logging_mixin.py:109} INFO - [2022-04-11 16:59:03,249] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:59:03,250] {logging_mixin.py:109} INFO - [2022-04-11 16:59:03,249] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = requests_cache.core.CachedSession(cache_name='cache', backend='sqlite')
AttributeError: module 'requests_cache' has no attribute 'core'
[2022-04-11 16:59:03,256] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:59:03,267] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.044 seconds
[2022-04-11 16:59:11,860] {processor.py:163} INFO - Started process (PID=13593) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:59:11,861] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:59:11,861] {logging_mixin.py:109} INFO - [2022-04-11 16:59:11,861] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:59:14,113] {logging_mixin.py:109} INFO - [2022-04-11 16:59:14,113] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:59:14,114] {logging_mixin.py:109} INFO - [2022-04-11 16:59:14,113] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = requests_cache.core.CachedSession(cache_name='cache', backend='sqlite')
AttributeError: module 'requests_cache' has no attribute 'core'
[2022-04-11 16:59:14,121] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:59:14,132] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.274 seconds
[2022-04-11 16:59:28,844] {processor.py:163} INFO - Started process (PID=13669) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:59:28,845] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:59:28,846] {logging_mixin.py:109} INFO - [2022-04-11 16:59:28,845] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:59:31,485] {logging_mixin.py:109} INFO - [2022-04-11 16:59:31,485] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:59:31,486] {logging_mixin.py:109} INFO - [2022-04-11 16:59:31,485] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    /
AttributeError: module 'requests_cache' has no attribute 'core'
[2022-04-11 16:59:31,492] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:59:31,503] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.662 seconds
[2022-04-11 16:59:52,146] {processor.py:163} INFO - Started process (PID=13753) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:59:52,148] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 16:59:52,149] {logging_mixin.py:109} INFO - [2022-04-11 16:59:52,148] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:59:54,976] {logging_mixin.py:109} INFO - [2022-04-11 16:59:54,976] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 16:59:54,977] {logging_mixin.py:109} INFO - [2022-04-11 16:59:54,976] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = requests_cache.core.CachedSession(cache_name='cache', backend='sqlite')
AttributeError: module 'requests_cache' has no attribute 'core'
[2022-04-11 16:59:54,984] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 16:59:54,996] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.853 seconds
[2022-04-11 17:00:10,387] {processor.py:163} INFO - Started process (PID=13766) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:00:10,389] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:00:10,394] {logging_mixin.py:109} INFO - [2022-04-11 17:00:10,394] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:00:16,228] {logging_mixin.py:109} INFO - [2022-04-11 17:00:16,228] {__init__.py:20} WARNING - cannot import name 'Counter' from 'cattr._compat' (/home/airflow/airflow/env/lib/python3.8/site-packages/cattr/_compat.py)
[2022-04-11 17:00:16,229] {logging_mixin.py:109} INFO - [2022-04-11 17:00:16,228] {dagbag.py:334} ERROR - Failed to import: /home/airflow/airflow/dags/dag_IS3107.py
Traceback (most recent call last):
  File "/home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/airflow/dags/dag_IS3107.py", line 18, in <module>
    session = Session()
NameError: name 'Session' is not defined
[2022-04-11 17:00:16,237] {processor.py:656} WARNING - No viable dags retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:00:16,249] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 5.870 seconds
[2022-04-11 17:00:45,774] {processor.py:163} INFO - Started process (PID=13923) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:00:45,777] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:00:45,778] {logging_mixin.py:109} INFO - [2022-04-11 17:00:45,777] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:00:52,664] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:00:53,832] {logging_mixin.py:109} INFO - [2022-04-11 17:00:53,832] {manager.py:496} INFO - Created Permission View: can read on DAG:IS3107_final
[2022-04-11 17:00:53,868] {logging_mixin.py:109} INFO - [2022-04-11 17:00:53,868] {manager.py:496} INFO - Created Permission View: can edit on DAG:IS3107_final
[2022-04-11 17:00:53,869] {logging_mixin.py:109} INFO - [2022-04-11 17:00:53,869] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:00:53,882] {logging_mixin.py:109} INFO - [2022-04-11 17:00:53,881] {dag.py:2415} INFO - Creating ORM DAG for IS3107_final
[2022-04-11 17:00:53,889] {logging_mixin.py:109} INFO - [2022-04-11 17:00:53,889] {dag.py:2935} INFO - Setting next_dagrun for IS3107_final to 2022-04-10T00:00:00+00:00
[2022-04-11 17:00:53,960] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 8.190 seconds
[2022-04-11 17:01:06,893] {processor.py:163} INFO - Started process (PID=14066) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:01:06,896] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:01:06,897] {logging_mixin.py:109} INFO - [2022-04-11 17:01:06,896] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:01:12,451] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:01:12,460] {logging_mixin.py:109} INFO - [2022-04-11 17:01:12,460] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:01:12,500] {logging_mixin.py:109} INFO - [2022-04-11 17:01:12,500] {dag.py:2935} INFO - Setting next_dagrun for IS3107_final to 2022-04-10T00:00:00+00:00
[2022-04-11 17:01:12,532] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 5.645 seconds
[2022-04-11 17:02:07,659] {processor.py:163} INFO - Started process (PID=14666) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:02:07,660] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:02:07,661] {logging_mixin.py:109} INFO - [2022-04-11 17:02:07,661] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:02:10,786] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:02:10,800] {logging_mixin.py:109} INFO - [2022-04-11 17:02:10,800] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:02:10,826] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.171 seconds
[2022-04-11 17:02:43,531] {processor.py:163} INFO - Started process (PID=15024) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:02:43,533] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:02:43,533] {logging_mixin.py:109} INFO - [2022-04-11 17:02:43,533] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:02:46,807] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:02:46,818] {logging_mixin.py:109} INFO - [2022-04-11 17:02:46,818] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:02:46,850] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.323 seconds
[2022-04-11 17:03:50,325] {processor.py:163} INFO - Started process (PID=15554) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:03:50,328] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:03:50,329] {logging_mixin.py:109} INFO - [2022-04-11 17:03:50,329] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:03:54,520] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:03:54,542] {logging_mixin.py:109} INFO - [2022-04-11 17:03:54,542] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:03:54,646] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 4.325 seconds
[2022-04-11 17:04:49,946] {processor.py:163} INFO - Started process (PID=15904) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:04:49,948] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:04:49,948] {logging_mixin.py:109} INFO - [2022-04-11 17:04:49,948] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:04:52,374] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:04:52,382] {logging_mixin.py:109} INFO - [2022-04-11 17:04:52,382] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:04:52,394] {logging_mixin.py:109} INFO - [2022-04-11 17:04:52,394] {dag.py:2935} INFO - Setting next_dagrun for IS3107_final to 2022-04-10T00:16:00+00:00
[2022-04-11 17:04:52,406] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.462 seconds
[2022-04-11 17:05:52,351] {processor.py:163} INFO - Started process (PID=16101) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:05:52,354] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:05:52,355] {logging_mixin.py:109} INFO - [2022-04-11 17:05:52,355] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:05:57,428] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:05:57,453] {logging_mixin.py:109} INFO - [2022-04-11 17:05:57,453] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:05:57,500] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 5.158 seconds
[2022-04-11 17:06:40,311] {processor.py:163} INFO - Started process (PID=16266) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:06:40,314] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:06:40,314] {logging_mixin.py:109} INFO - [2022-04-11 17:06:40,314] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:06:43,379] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:06:43,403] {logging_mixin.py:109} INFO - [2022-04-11 17:06:43,401] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:06:43,460] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.152 seconds
[2022-04-11 17:07:18,197] {processor.py:163} INFO - Started process (PID=16440) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:07:18,200] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:07:18,201] {logging_mixin.py:109} INFO - [2022-04-11 17:07:18,201] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:07:23,090] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:07:23,113] {logging_mixin.py:109} INFO - [2022-04-11 17:07:23,112] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:07:23,160] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 4.969 seconds
[2022-04-11 17:07:54,963] {processor.py:163} INFO - Started process (PID=16522) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:07:54,967] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:07:54,968] {logging_mixin.py:109} INFO - [2022-04-11 17:07:54,968] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:08:00,283] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:08:00,311] {logging_mixin.py:109} INFO - [2022-04-11 17:08:00,310] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:08:00,387] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 5.432 seconds
[2022-04-11 17:08:32,412] {processor.py:163} INFO - Started process (PID=16588) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:08:32,419] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:08:32,420] {logging_mixin.py:109} INFO - [2022-04-11 17:08:32,420] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:08:37,254] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:08:37,264] {logging_mixin.py:109} INFO - [2022-04-11 17:08:37,263] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:08:37,288] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 4.888 seconds
[2022-04-11 17:09:29,813] {processor.py:163} INFO - Started process (PID=16762) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:09:29,814] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:09:29,815] {logging_mixin.py:109} INFO - [2022-04-11 17:09:29,815] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:09:32,827] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:09:32,838] {logging_mixin.py:109} INFO - [2022-04-11 17:09:32,838] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:09:32,865] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.055 seconds
[2022-04-11 17:10:25,968] {processor.py:163} INFO - Started process (PID=17039) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:10:25,970] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:10:25,971] {logging_mixin.py:109} INFO - [2022-04-11 17:10:25,971] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:10:29,849] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:10:29,862] {logging_mixin.py:109} INFO - [2022-04-11 17:10:29,862] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:10:29,890] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.926 seconds
[2022-04-11 17:11:16,322] {processor.py:163} INFO - Started process (PID=17204) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:11:16,324] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:11:16,325] {logging_mixin.py:109} INFO - [2022-04-11 17:11:16,324] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:11:18,569] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:11:18,578] {logging_mixin.py:109} INFO - [2022-04-11 17:11:18,578] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:11:18,601] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.282 seconds
[2022-04-11 17:11:58,362] {processor.py:163} INFO - Started process (PID=17433) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:11:58,363] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:11:58,363] {logging_mixin.py:109} INFO - [2022-04-11 17:11:58,363] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:12:03,759] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:12:03,769] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:12:03,786] {logging_mixin.py:109} INFO - [2022-04-11 17:12:03,781] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:12:03,816] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:12:03,858] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 5.504 seconds
[2022-04-11 17:13:07,817] {processor.py:163} INFO - Started process (PID=17837) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:13:07,818] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:13:07,820] {logging_mixin.py:109} INFO - [2022-04-11 17:13:07,819] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:13:12,406] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:13:12,413] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:13:12,424] {logging_mixin.py:109} INFO - [2022-04-11 17:13:12,424] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:13:12,439] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:13:12,464] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 4.653 seconds
[2022-04-11 17:14:06,318] {processor.py:163} INFO - Started process (PID=18003) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:14:06,319] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:14:06,321] {logging_mixin.py:109} INFO - [2022-04-11 17:14:06,320] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:14:09,521] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:14:09,528] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:14:09,538] {logging_mixin.py:109} INFO - [2022-04-11 17:14:09,538] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:14:09,552] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:14:09,584] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.277 seconds
[2022-04-11 17:14:53,214] {processor.py:163} INFO - Started process (PID=18201) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:14:53,215] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:14:53,215] {logging_mixin.py:109} INFO - [2022-04-11 17:14:53,215] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:14:55,813] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:14:55,821] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:14:55,835] {logging_mixin.py:109} INFO - [2022-04-11 17:14:55,834] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:14:55,857] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:14:55,889] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.678 seconds
[2022-04-11 17:15:40,090] {processor.py:163} INFO - Started process (PID=18305) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:15:40,091] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:15:40,091] {logging_mixin.py:109} INFO - [2022-04-11 17:15:40,091] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:15:42,259] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:15:42,265] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:15:42,275] {logging_mixin.py:109} INFO - [2022-04-11 17:15:42,274] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:15:42,288] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:15:42,313] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.226 seconds
[2022-04-11 17:16:27,912] {processor.py:163} INFO - Started process (PID=18473) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:16:27,914] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:16:27,915] {logging_mixin.py:109} INFO - [2022-04-11 17:16:27,915] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:16:31,064] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:16:31,072] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:16:31,086] {logging_mixin.py:109} INFO - [2022-04-11 17:16:31,085] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:16:31,106] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:16:31,147] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.243 seconds
[2022-04-11 17:17:15,301] {processor.py:163} INFO - Started process (PID=18648) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:17:15,302] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:17:15,303] {logging_mixin.py:109} INFO - [2022-04-11 17:17:15,303] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:17:18,329] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:17:18,336] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:17:18,350] {logging_mixin.py:109} INFO - [2022-04-11 17:17:18,350] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:17:18,369] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:17:18,400] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.102 seconds
[2022-04-11 17:17:52,212] {processor.py:163} INFO - Started process (PID=18729) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:17:52,213] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:17:52,214] {logging_mixin.py:109} INFO - [2022-04-11 17:17:52,214] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:17:54,655] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:17:54,662] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:17:54,672] {logging_mixin.py:109} INFO - [2022-04-11 17:17:54,671] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:17:54,686] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:17:54,713] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.504 seconds
[2022-04-11 17:21:57,666] {processor.py:163} INFO - Started process (PID=19041) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:21:57,667] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:21:57,668] {logging_mixin.py:109} INFO - [2022-04-11 17:21:57,668] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:22:00,220] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:22:00,228] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:22:00,239] {logging_mixin.py:109} INFO - [2022-04-11 17:22:00,239] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:22:00,256] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:22:00,290] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.627 seconds
[2022-04-11 17:23:12,965] {processor.py:163} INFO - Started process (PID=19482) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:23:12,966] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:23:12,966] {logging_mixin.py:109} INFO - [2022-04-11 17:23:12,966] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:23:17,342] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:23:17,351] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:23:17,364] {logging_mixin.py:109} INFO - [2022-04-11 17:23:17,364] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:23:17,386] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:23:17,422] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 4.462 seconds
[2022-04-11 17:24:04,179] {processor.py:163} INFO - Started process (PID=19515) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:24:04,179] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:24:04,180] {logging_mixin.py:109} INFO - [2022-04-11 17:24:04,180] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:24:07,042] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:24:07,049] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:24:07,059] {logging_mixin.py:109} INFO - [2022-04-11 17:24:07,059] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:24:07,073] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:24:07,104] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.929 seconds
[2022-04-11 17:24:46,556] {processor.py:163} INFO - Started process (PID=19544) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:24:46,558] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:24:46,559] {logging_mixin.py:109} INFO - [2022-04-11 17:24:46,559] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:24:48,685] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:24:48,693] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:24:48,708] {logging_mixin.py:109} INFO - [2022-04-11 17:24:48,707] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:24:48,726] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:24:48,759] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.205 seconds
[2022-04-11 17:25:24,727] {processor.py:163} INFO - Started process (PID=19570) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:25:24,728] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:25:24,728] {logging_mixin.py:109} INFO - [2022-04-11 17:25:24,728] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:25:28,533] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:25:28,541] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:25:28,558] {logging_mixin.py:109} INFO - [2022-04-11 17:25:28,558] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:25:28,578] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:25:28,614] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.891 seconds
[2022-04-11 17:26:14,866] {processor.py:163} INFO - Started process (PID=19602) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:26:14,869] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:26:14,869] {logging_mixin.py:109} INFO - [2022-04-11 17:26:14,869] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:26:17,300] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:26:17,311] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:26:17,329] {logging_mixin.py:109} INFO - [2022-04-11 17:26:17,329] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:26:17,351] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:26:17,382] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.519 seconds
[2022-04-11 17:27:04,885] {processor.py:163} INFO - Started process (PID=19808) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:27:04,886] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:27:04,887] {logging_mixin.py:109} INFO - [2022-04-11 17:27:04,887] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:27:08,571] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:27:08,584] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:27:08,606] {logging_mixin.py:109} INFO - [2022-04-11 17:27:08,605] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:27:08,651] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:27:08,708] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.827 seconds
[2022-04-11 17:27:57,872] {processor.py:163} INFO - Started process (PID=19854) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:27:57,874] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:27:57,875] {logging_mixin.py:109} INFO - [2022-04-11 17:27:57,875] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:28:00,303] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:28:00,310] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:28:00,321] {logging_mixin.py:109} INFO - [2022-04-11 17:28:00,320] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:28:00,337] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:28:00,362] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.494 seconds
[2022-04-11 17:28:38,680] {processor.py:163} INFO - Started process (PID=19882) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:28:38,681] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:28:38,681] {logging_mixin.py:109} INFO - [2022-04-11 17:28:38,681] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:28:42,605] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:28:42,614] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:28:42,630] {logging_mixin.py:109} INFO - [2022-04-11 17:28:42,630] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:28:42,651] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:28:42,685] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 4.008 seconds
[2022-04-11 17:29:21,740] {processor.py:163} INFO - Started process (PID=19911) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:29:21,742] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:29:21,743] {logging_mixin.py:109} INFO - [2022-04-11 17:29:21,743] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:29:24,389] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:29:24,396] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:29:24,406] {logging_mixin.py:109} INFO - [2022-04-11 17:29:24,405] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:29:24,420] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:29:24,445] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.710 seconds
[2022-04-11 17:30:02,653] {processor.py:163} INFO - Started process (PID=19939) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:30:02,654] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:30:02,655] {logging_mixin.py:109} INFO - [2022-04-11 17:30:02,655] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:30:05,127] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:30:05,134] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:30:05,145] {logging_mixin.py:109} INFO - [2022-04-11 17:30:05,145] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:30:05,161] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:30:05,191] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.542 seconds
[2022-04-11 17:30:42,438] {processor.py:163} INFO - Started process (PID=19966) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:30:42,439] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:30:42,440] {logging_mixin.py:109} INFO - [2022-04-11 17:30:42,439] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:30:45,358] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:30:45,364] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:30:45,373] {logging_mixin.py:109} INFO - [2022-04-11 17:30:45,373] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:30:45,388] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:30:45,412] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.978 seconds
[2022-04-11 17:31:25,165] {processor.py:163} INFO - Started process (PID=19995) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:31:25,165] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:31:25,166] {logging_mixin.py:109} INFO - [2022-04-11 17:31:25,166] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:31:28,437] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:31:28,452] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:31:28,465] {logging_mixin.py:109} INFO - [2022-04-11 17:31:28,465] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:31:28,483] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:31:28,513] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.353 seconds
[2022-04-11 17:32:03,093] {processor.py:163} INFO - Started process (PID=20114) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:32:03,095] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:32:03,095] {logging_mixin.py:109} INFO - [2022-04-11 17:32:03,095] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:32:05,310] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:32:05,316] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:32:05,325] {logging_mixin.py:109} INFO - [2022-04-11 17:32:05,325] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:32:05,338] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:32:05,363] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.273 seconds
[2022-04-11 17:32:43,738] {processor.py:163} INFO - Started process (PID=20238) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:32:43,739] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:32:43,739] {logging_mixin.py:109} INFO - [2022-04-11 17:32:43,739] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:32:46,283] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:32:46,291] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:32:46,303] {logging_mixin.py:109} INFO - [2022-04-11 17:32:46,303] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:32:46,322] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:32:46,358] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.626 seconds
[2022-04-11 17:33:23,837] {processor.py:163} INFO - Started process (PID=20313) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:33:23,839] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:33:23,840] {logging_mixin.py:109} INFO - [2022-04-11 17:33:23,839] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:33:25,974] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:33:25,980] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:33:25,989] {logging_mixin.py:109} INFO - [2022-04-11 17:33:25,989] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:33:26,002] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:33:26,026] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.192 seconds
[2022-04-11 17:34:06,419] {processor.py:163} INFO - Started process (PID=20364) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:34:06,419] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:34:06,420] {logging_mixin.py:109} INFO - [2022-04-11 17:34:06,420] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:34:08,573] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:34:08,582] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:34:08,591] {logging_mixin.py:109} INFO - [2022-04-11 17:34:08,590] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:34:08,604] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:34:08,628] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.213 seconds
[2022-04-11 17:34:42,718] {processor.py:163} INFO - Started process (PID=20455) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:34:42,719] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:34:42,719] {logging_mixin.py:109} INFO - [2022-04-11 17:34:42,719] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:34:44,464] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:34:44,468] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:34:44,475] {logging_mixin.py:109} INFO - [2022-04-11 17:34:44,475] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:34:44,484] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:34:44,502] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 1.786 seconds
[2022-04-11 17:35:26,751] {processor.py:163} INFO - Started process (PID=20483) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:35:26,751] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:35:26,752] {logging_mixin.py:109} INFO - [2022-04-11 17:35:26,752] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:35:28,989] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:35:28,994] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:35:29,002] {logging_mixin.py:109} INFO - [2022-04-11 17:35:29,002] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:35:29,014] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:35:29,036] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.288 seconds
[2022-04-11 17:36:09,109] {processor.py:163} INFO - Started process (PID=20512) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:36:09,110] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:36:09,110] {logging_mixin.py:109} INFO - [2022-04-11 17:36:09,110] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:36:12,079] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:36:12,085] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:36:12,094] {logging_mixin.py:109} INFO - [2022-04-11 17:36:12,093] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:36:12,106] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:36:12,128] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.024 seconds
[2022-04-11 17:37:03,823] {processor.py:163} INFO - Started process (PID=20965) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:37:03,826] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:37:03,827] {logging_mixin.py:109} INFO - [2022-04-11 17:37:03,827] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:37:08,747] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:37:08,757] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:37:08,796] {logging_mixin.py:109} INFO - [2022-04-11 17:37:08,795] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:37:08,817] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:37:08,881] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 5.065 seconds
[2022-04-11 17:37:43,524] {processor.py:163} INFO - Started process (PID=21005) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:37:43,530] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:37:43,531] {logging_mixin.py:109} INFO - [2022-04-11 17:37:43,531] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:37:50,350] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:37:50,361] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:37:50,380] {logging_mixin.py:109} INFO - [2022-04-11 17:37:50,379] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:37:50,410] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:37:50,459] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 6.943 seconds
[2022-04-11 17:38:26,211] {processor.py:163} INFO - Started process (PID=21024) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:38:26,211] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:38:26,212] {logging_mixin.py:109} INFO - [2022-04-11 17:38:26,212] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:38:30,957] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:38:30,971] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:38:30,988] {logging_mixin.py:109} INFO - [2022-04-11 17:38:30,988] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:38:31,018] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:38:31,058] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 4.869 seconds
[2022-04-11 17:39:07,270] {processor.py:163} INFO - Started process (PID=21153) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:39:07,273] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:39:07,274] {logging_mixin.py:109} INFO - [2022-04-11 17:39:07,274] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:39:10,313] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:39:10,320] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:39:10,329] {logging_mixin.py:109} INFO - [2022-04-11 17:39:10,329] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:39:10,343] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:39:10,368] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.104 seconds
[2022-04-11 17:40:00,502] {processor.py:163} INFO - Started process (PID=21232) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:40:00,504] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:40:00,505] {logging_mixin.py:109} INFO - [2022-04-11 17:40:00,505] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:40:03,336] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:40:03,344] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:40:03,359] {logging_mixin.py:109} INFO - [2022-04-11 17:40:03,359] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:40:03,386] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:40:03,438] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.939 seconds
[2022-04-11 17:40:53,183] {processor.py:163} INFO - Started process (PID=21306) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:40:53,185] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:40:53,186] {logging_mixin.py:109} INFO - [2022-04-11 17:40:53,186] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:40:57,625] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:40:57,632] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:40:57,645] {logging_mixin.py:109} INFO - [2022-04-11 17:40:57,644] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:40:57,662] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:40:57,716] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 4.542 seconds
[2022-04-11 17:41:45,292] {processor.py:163} INFO - Started process (PID=21484) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:41:45,294] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:41:45,295] {logging_mixin.py:109} INFO - [2022-04-11 17:41:45,295] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:41:48,307] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:41:48,313] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:41:48,322] {logging_mixin.py:109} INFO - [2022-04-11 17:41:48,322] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:41:48,336] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:41:48,363] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.076 seconds
[2022-04-11 17:42:38,624] {processor.py:163} INFO - Started process (PID=21830) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:42:38,626] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:42:38,628] {logging_mixin.py:109} INFO - [2022-04-11 17:42:38,627] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:42:43,698] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:42:43,706] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:42:43,721] {logging_mixin.py:109} INFO - [2022-04-11 17:42:43,721] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:42:43,746] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:42:43,816] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 5.198 seconds
[2022-04-11 17:43:38,919] {processor.py:163} INFO - Started process (PID=22027) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:43:38,920] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:43:38,920] {logging_mixin.py:109} INFO - [2022-04-11 17:43:38,920] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:43:41,343] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:43:41,348] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:43:41,358] {logging_mixin.py:109} INFO - [2022-04-11 17:43:41,358] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:43:41,371] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:43:41,405] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.489 seconds
[2022-04-11 17:44:24,098] {processor.py:163} INFO - Started process (PID=22284) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:44:24,099] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:44:24,100] {logging_mixin.py:109} INFO - [2022-04-11 17:44:24,099] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:44:27,337] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:44:27,345] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:44:27,358] {logging_mixin.py:109} INFO - [2022-04-11 17:44:27,357] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:44:27,373] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:44:27,419] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.325 seconds
[2022-04-11 17:45:17,405] {processor.py:163} INFO - Started process (PID=22472) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:45:17,407] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:45:17,407] {logging_mixin.py:109} INFO - [2022-04-11 17:45:17,407] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:45:21,702] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:45:21,713] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:45:21,732] {logging_mixin.py:109} INFO - [2022-04-11 17:45:21,731] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:45:21,764] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:45:21,816] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 4.414 seconds
[2022-04-11 17:46:29,636] {processor.py:163} INFO - Started process (PID=22860) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:46:29,641] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:46:29,642] {logging_mixin.py:109} INFO - [2022-04-11 17:46:29,642] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:46:39,935] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:46:39,950] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:46:39,989] {logging_mixin.py:109} INFO - [2022-04-11 17:46:39,989] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:46:40,062] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:46:40,139] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 10.520 seconds
[2022-04-11 17:47:45,632] {processor.py:163} INFO - Started process (PID=23251) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:47:45,638] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:47:45,639] {logging_mixin.py:109} INFO - [2022-04-11 17:47:45,638] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:47:51,157] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:47:51,172] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:47:51,190] {logging_mixin.py:109} INFO - [2022-04-11 17:47:51,190] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:47:51,230] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:47:51,271] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 5.644 seconds
[2022-04-11 17:48:33,361] {processor.py:163} INFO - Started process (PID=23348) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:48:33,362] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:48:33,362] {logging_mixin.py:109} INFO - [2022-04-11 17:48:33,362] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:48:39,561] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:48:39,598] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:48:39,617] {logging_mixin.py:109} INFO - [2022-04-11 17:48:39,617] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:48:39,651] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:48:39,733] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 6.375 seconds
[2022-04-11 17:49:19,829] {processor.py:163} INFO - Started process (PID=23612) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:49:19,834] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:49:19,834] {logging_mixin.py:109} INFO - [2022-04-11 17:49:19,834] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:49:24,574] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:49:24,582] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:49:24,598] {logging_mixin.py:109} INFO - [2022-04-11 17:49:24,598] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:49:24,617] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:49:24,665] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 4.839 seconds
[2022-04-11 17:50:13,029] {processor.py:163} INFO - Started process (PID=23864) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:50:13,032] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:50:13,032] {logging_mixin.py:109} INFO - [2022-04-11 17:50:13,032] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:50:14,946] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:50:14,952] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:50:14,962] {logging_mixin.py:109} INFO - [2022-04-11 17:50:14,962] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:50:14,976] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:50:15,000] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 1.973 seconds
[2022-04-11 17:50:49,905] {processor.py:163} INFO - Started process (PID=24076) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:50:49,906] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:50:49,915] {logging_mixin.py:109} INFO - [2022-04-11 17:50:49,915] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:50:53,301] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:50:53,308] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:50:53,318] {logging_mixin.py:109} INFO - [2022-04-11 17:50:53,318] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:50:53,333] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:50:53,367] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.465 seconds
[2022-04-11 17:51:39,333] {processor.py:163} INFO - Started process (PID=24139) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:51:39,333] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:51:39,334] {logging_mixin.py:109} INFO - [2022-04-11 17:51:39,334] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:51:41,658] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:51:41,665] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:51:41,676] {logging_mixin.py:109} INFO - [2022-04-11 17:51:41,676] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:51:41,692] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:51:41,721] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.391 seconds
[2022-04-11 17:52:23,770] {processor.py:163} INFO - Started process (PID=24224) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:52:23,772] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:52:23,774] {logging_mixin.py:109} INFO - [2022-04-11 17:52:23,773] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:52:27,718] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:52:27,726] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:52:27,737] {logging_mixin.py:109} INFO - [2022-04-11 17:52:27,736] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:52:27,752] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:52:27,778] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 4.014 seconds
[2022-04-11 17:53:08,503] {processor.py:163} INFO - Started process (PID=24406) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:53:08,504] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:53:08,506] {logging_mixin.py:109} INFO - [2022-04-11 17:53:08,506] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:53:10,804] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:53:10,809] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:53:10,816] {logging_mixin.py:109} INFO - [2022-04-11 17:53:10,816] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:53:10,827] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:53:10,847] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.348 seconds
[2022-04-11 17:53:48,862] {processor.py:163} INFO - Started process (PID=24547) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:53:48,866] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:53:48,866] {logging_mixin.py:109} INFO - [2022-04-11 17:53:48,866] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:53:52,582] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:53:52,587] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:53:52,594] {logging_mixin.py:109} INFO - [2022-04-11 17:53:52,594] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:53:52,605] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:53:52,626] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.768 seconds
[2022-04-11 17:54:29,336] {processor.py:163} INFO - Started process (PID=24668) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:54:29,338] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:54:29,341] {logging_mixin.py:109} INFO - [2022-04-11 17:54:29,340] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:54:32,184] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:54:32,189] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:54:32,197] {logging_mixin.py:109} INFO - [2022-04-11 17:54:32,197] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:54:32,209] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:54:32,230] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.902 seconds
[2022-04-11 17:55:14,032] {processor.py:163} INFO - Started process (PID=24763) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:55:14,034] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:55:14,035] {logging_mixin.py:109} INFO - [2022-04-11 17:55:14,035] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:55:18,402] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:55:18,412] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:55:18,430] {logging_mixin.py:109} INFO - [2022-04-11 17:55:18,430] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:55:18,452] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:55:18,489] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 4.463 seconds
[2022-04-11 17:55:58,222] {processor.py:163} INFO - Started process (PID=24867) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:55:58,223] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:55:58,223] {logging_mixin.py:109} INFO - [2022-04-11 17:55:58,223] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:56:02,010] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:56:02,017] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:56:02,028] {logging_mixin.py:109} INFO - [2022-04-11 17:56:02,027] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:56:02,043] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:56:02,073] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 3.854 seconds
[2022-04-11 17:56:50,090] {processor.py:163} INFO - Started process (PID=25074) to work on /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:56:50,091] {processor.py:642} INFO - Processing file /home/airflow/airflow/dags/dag_IS3107.py for tasks to queue
[2022-04-11 17:56:50,091] {logging_mixin.py:109} INFO - [2022-04-11 17:56:50,091] {dagbag.py:500} INFO - Filling up the DagBag from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:56:52,130] {processor.py:654} INFO - DAG(s) dict_keys(['IS3107_final']) retrieved from /home/airflow/airflow/dags/dag_IS3107.py
[2022-04-11 17:56:52,136] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:56:52,148] {logging_mixin.py:109} INFO - [2022-04-11 17:56:52,148] {dag.py:2396} INFO - Sync 1 DAGs
[2022-04-11 17:56:52,166] {logging_mixin.py:109} WARNING - /home/airflow/airflow/env/lib/python3.8/site-packages/airflow/models/dag.py:2433 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-04-11 17:56:52,207] {processor.py:171} INFO - Processing /home/airflow/airflow/dags/dag_IS3107.py took 2.119 seconds
